### ISSUE TITLE
Reply, Delete, New mail buttons workflow

### ISSUE THREAD

**Issue by 2Belette** [scoute-dich/K9-MailClient#25]
Hi,

I like the look and feel of this fork but I am not sure I am understanding the philosophy to use it?
I don't see the buttons bar like in K9, so I don't see another alternative to reply as to push to an email from the email list and get the menu with the options.
Is there any other way to reply when I open an email or should I always go back to the emails list?
I am sure I am missing a logic somewhere..
Many thanks!

**Comment by PanderMusubi** [comment]
When scrolling down in a folder, eventually seeing a + icon appear for composing a new email would be nice in the bottom right corner.

Similarly, when scrolling down in a message, a reply icon may appear.

**Comment by lindhe** [comment]
I agree — it's odd not to have reply or reply all readily available.
